samples/apps/copilot-chat-app/webapp/src/components/chat/chat-history/ChatHistoryDocumentContent.tsx

```diff
@@ -54,40 +59,48 @@ interface ChatHistoryDocumentContentProps {
     message: IChatMessage;
 }
 
-interface DocumentMessageContent {
+interface DocumentData {
     name: string;
     size: string;
+    status: boolean;
+}
+
+interface DocumentMessageContent {
+    documents: DocumentData[];
 }
 
 export const ChatHistoryDocumentContent: React.FC<ChatHistoryDocumentContentProps> = ({ isMe, message }) => {
     const classes = useClasses();
 
-    let name = '',
-        size = '';
+    let documents: DocumentData[] = [];
     try {
-        ({ name, size } = JSON.parse(message.content) as DocumentMessageContent);
+        ({ documents } = JSON.parse(message.content) as DocumentMessageContent);
     } catch (e) {
         console.error('Error parsing chat history file item: ' + message.content);
     }
 
     return (
         <>
-            <Card appearance="filled-alternative" className={classes.card}>
-                <CardHeader
-                    className={classes.cardHeader}
-                    image={getFileIconByFileExtension(name, { className: classes.icon })}
-                    header={<Text className={classes.cardHeaderText}>{name}</Text>}
-                    description={
-                        <Caption1 block className={classes.cardCaption}>
-                            {size}
-                        </Caption1>
-                    }
-                />
-                <ProgressBar thickness="large" color="success" value={1} />
-            </Card>
-            <span className={isMe ? classes.footer : mergeClasses(classes.footer, classes.floatLeft)}>
-                Success: memory established
-            </span>
+            {documents.map((document, index) => (
+                <div className={classes.root} key={`${message.id}-document-${index}`}>
+                    <Card appearance="filled-alternative" className={classes.card}>
+                        <CardHeader
+                            className={classes.cardHeader}
+                            image={getFileIconByFileExtension(document.name, { className: classes.icon })}
+                            header={<Text className={classes.cardHeaderText}>{document.name}</Text>}
+                            description={
+                                <Caption1 block className={classes.cardCaption}>
+                                    {document.size}
+                                </Caption1>
+                            }
+                        />
+                        <ProgressBar thickness="large" color={document.status ? "success" : "error"} value={1} />
+                    </Card>
+                    <span className={isMe ? classes.footer : mergeClasses(classes.footer, classes.floatLeft)}>
+                        {document.status ? "Success: memory established" : "Failed: memory not established"}
```

Comment:
Is there a better error message we can return here? If it's not established, is there a readme or docs site we can point the user to?

Reply:
If the upload is unsuccessful, an error message (alert) will appear.

Reply:
> @TaoChenOSU it may be good to check with Ruokan to see if she has thoughts on the pattern here

Will do. Maybe we can change the design in another PR